pages/people.md

```diff
@@ -48,6 +48,7 @@ based on commits.
 | [Shan Nawaz](https://github.com/shannawaz)                  | [Microsoft][msft]                   | [ET][et]   |
 | [Sri Kailash](https://github.com/srikailash)                | [Booking.com](https://booking.com)  | [CEST][cst]|
 | [Tanisha Faulkner](https://github.com/plainolneesh)         | [ASRC Federal](https://www.asrc.com)| [ET][et]   |
+| [Ben Kelly](https://github.com/benkelly)         | [Microsoft][msft] | [GMT][gmt]   |
```

Comment:
I think this should be aligned with the others, and I think this list is alphabetical, too.
If you can fix those, I can merge and update the website pretty quickly after that.

Reply:
Whoops, I didn't notice I was in alphabetical order, fixed now!